modules/bookClass.js

```diff
@@ -0,0 +1,66 @@
+import { bookContainer } from './htmlElements.js';
+import Collections from './collectionsClass.js';
+
+const collections = new Collections();
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.timeStamp = Book.timeStamp();
+  }
+
+  addBook() {
+    collections.addBook(this);
+    this.render();
+  }
```

Comment:
Great job @betses 🤝

- [ ] As specified in the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md), kindly format all your methods with ES6 syntax across all your JS files.